coderd/database/pubsub/pubsub_test.go

```diff
@@ -119,3 +120,79 @@ func TestPGPubsub_Metrics(t *testing.T) {
 			!testutil.PromCounterGathered(t, metrics, "coder_pubsub_latency_measure_errs_total")
 	}, testutil.WaitShort, testutil.IntervalFast)
 }
+
+func TestPGPubsubDriver(t *testing.T) {
```

Comment:
This is a good test, but the design of the instrumented driver is racy -- see comments on `dbtestutil/driver.go` for how to address.